README.md

```diff
@@ -0,0 +1,78 @@
+# secret-script
+Proof-of-concept python script for sychronization between kubernetes secret and gcloud secret manager.
+
+## Prerequisites
+- Create a gke cluster.
+
+- [Create service account for app](https://cloud.google.com/docs/authentication/production#command-line)
+
+- Grant required permissions to the service account `service-account-name`.
+
+	- Permission to get clusters:
+
+		    gcloud projects add-iam-policy-binding k8s-jkns-gke-soak --member "serviceAccount:<service-account-name>@k8s-jkns-gke-soak.iam.gserviceaccount.com" --role "roles/container.clusterViewer"
+	- Permission to manage secrets:
+
+		- Create a custom iam role `iam-role-id` with container.secrets.* permissions and add the role to service account `service-account-name`:
+			- service-sevret-role.yaml
```

Comment:
```suggestion
			- service-secret-role.yaml
```

Reply:
consider including this file in the repo

Reply:
Added. Though in this demo I chose the other method (roles/container.developer), because I do not have the permission to add a custom role.